### PR TITLE
ACMS-1073: Robust Capabilities

### PR DESCRIPTION
**Pull request by emb03**
**Motivation**

Adds new module that provides a low-barrier of entry to get up and running with a NEXTjs site by automating many steps required to get a site working.

**Proposed changes**
• When installing Robust module, create a user named "Headless" with a role of "headless" and a customized consumer is created
• Customized consumer displays: User, Description, UID, secret, Scopes and Operations

**Testing steps**
 Install a new site
• Enable the ACMS Headless module and Robust Headless module
• Go to the consumer page, new consumer should be appear
• Go to next entity page, next site should appear
• Go to consumer edit page User (Headless) and Scope (headless) should be filled out and editable.
• Go to admin/people a new user should be there with the name of "Headless" and role of "headless".
• Uninstall the robust module, go to admin/people user (Headless) with role (headless) should have remained
• Go to consumer page, consumer should remain
• Go to next entity page, next site should remain
• Re-install Headless Robust module, you should not get errors

**Merge requirements**
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Comment by emb03** [comment]
@modularlab Just pushed updated file, forgot to stage it. sorry!

**Comment by emb03** [comment]
@modularlab pushed another commit.

**Comment by emb03** [comment]
@modularlab new commit, please review

File: modules/acquia_cms_headless/modules/acquia_cms_headless_robustapi/src/HeadlessRobustApiInstallHandler.php [hunk 1]
```diff
@@ -106,7 +105,7 @@ public function createHeadlessConsumer() {
         // consumer secret before it's stored in a hash key.
         $this->messenger->addStatus($this->t('Your Oauth consumer secret has been generated for you and is: <h2>@secret</h2> Please store this value as it cannot be retrieved. Alternatively you can generate a new secret via the <a href="@link">Consumer UI</a>', [
           '@secret' => $secret,
-          '@link' => Url::fromRoute('consumers.admin.config.services'),
+          '@link' => ('config/services/consumer'),
```
Comment (modularlab) [inline modules/acquia_cms_headless/modules/acquia_cms_headless_robustapi/src/HeadlessRobustApiInstallHandler.php]: I was hoping to use a link function here.  such as `URL()`.  Regardless, shouldn't it be `/admin/config/services/consumer`?

Comment (emb03) [reply modules/acquia_cms_headless/modules/acquia_cms_headless_robustapi/src/HeadlessRobustApiInstallHandler.php]: I tried this options:
URL::fromURI(internal:...) internal and it still called admin/modules (is that what you mean?)
putting /admin in the link generates a link: admin/admin....

Comment (modularlab) [reply modules/acquia_cms_headless/modules/acquia_cms_headless_robustapi/src/HeadlessRobustApiInstallHandler.php]: Odd.  Okay.  We can leave it as is for now then if it's working.

Comment (emb03) [reply modules/acquia_cms_headless/modules/acquia_cms_headless_robustapi/src/HeadlessRobustApiInstallHandler.php]: it was a last-ditch effort it seemed weird to me that it worked.

**Comment by emb03** [comment]
@lwalker-bounteous Here is the README file if you want to take a gander.

**Pull request closed without merging**